src/components/homepage/DownloadBanner.astro

```diff
@@ -7,8 +7,8 @@ import DownloadMuseHubButton from "../button/DownloadMuseHubButton";
   <div class="max-w-screen md:max-w-screen-lg mx-6 lg:mx-auto text-center">
     <h2>Audacity is free, forever</h2>
     <div class="flex flex-col sm:flex-row gap-4 mt-4 justify-center">
-      <DownloadButton client:only />
-      <DownloadMuseHubButton client:only />
+      <DownloadButton client:load />
+      <DownloadMuseHubButton client:load />
```

Comment:
given that these guys are far below the fold and rather small, doing a `client:visible` may be appropriate here as well.